NodeManager.cpp

```diff
@@ -1040,6 +1040,87 @@ void SensorDHT::onReceive(const MyMessage & message) {
 #endif
 
 /*
+   SensorAM2320
+*/
+#if MODULE_AM2320 == 1
+// contructor
+SensorAM2320::SensorAM2320(int child_id, int sensor_type): Sensor(child_id,A2) {
+  _th = new AM2320(); //another approach to try
```

Comment:
It ok, it works as you first suggested, I will let you check in the soon to come PR.